netlab/lib/Screens/Home/home.dart

```diff
@@ -25,6 +25,12 @@ class Home extends StatelessWidget {
                   Navigator.of(context).pushNamed('/joinroom');
                 },
               ),
+              RaisedButton(
```

Comment:
Utilise le widget CustomButton ici, si tu regarde le [home.dart](https://github.com/Un-dev/Netlab/blob/dev-app/netlab/lib/Screens/Home/home.dart) de la branch dev-app j'ai utilisé CustomButton partout
```suggestion
              CustomButton(
```

Reply:
J'ai remplacé RaisedButton par CustomButtom ICI